assessmentModel/src/commonMain/kotlin/Assessment.kt

```diff
@@ -394,7 +394,18 @@ interface CountdownStep : OptionalStep, ActiveStep
  * For example, a [FormStep] may describe entering a participant's demographics data where the study designer wants to
  * display height, weight, gender, and birth year on a single screen.
  */
-interface FormStep : Step, NodeContainer, ContentNode
+interface FormStep : Step, ContentNode {
```

Comment:
Changed from implementing NodeContainer b/c the NodeContainer became an inheritance pattern for a branch node (which is intended for navigation within a set of nodes) whereas the purpose of a form step is to allow showing a collection of nodes in a single view.